pipeline/alignments/align.py

```diff
@@ -438,14 +438,21 @@ def main() -> None:
     )
     args = parser.parse_args()
     logger.info("Starting generating alignments.")
+
+    priors_input_path = args.priors_input_path
+    if priors_input_path and not Path(priors_input_path).exists():
+        # This can happen on training continuation.
+        print("The priors were not found, they will be regenerated.")
+        priors_input_path = None
```

Comment:
As discussed, we should test how it works on high-resource languages later. Using priors added some kind of optimization to memory usage.

Reply:
Maybe it's worth adding a comment or todo on what we plan to do about priors in the future.